### DISCUSSION TITLE
Why is my installed winget version higher than the version I can build from this repo?

### DISCUSSION THREAD

**Discussion by ev-dev** [microsoft/winget-cli#1792] [Q&A] [answered]
Hey bit of a strange question but I was trying to debug and work on a winget issue but when I went to build the solution from master branch, I found out that my machine's winget version was a whole minor version ahead of what I could build from source, so I was wondering how that's possible and how to access the source of the working tree to build locally.

I'm running WIndows 11 insider beta channel, my pre-installed winget details are:
- Windows Package Manager (Preview) v1.2.3411-preview
- Windows: Windows.Desktop v10.0.22000.348
- Package: Microsoft.DesktopAppInstaller v1.17.3411.0

Whereas building the Debug x64 solution on master branch produces a v1.1.x winget. The repo itself doesn't contain a lot of docs as far as building so any details would be really appreciated, and thanks to all contributors!

**Comment by jedieaston** [comment, marked as answer]
They push builds to Insiders before adding releases to GitHub, as winget-cli is bundled with the closed source App Installer app (which is a GUI for installing msix/appx packages). Also, the winget that Visual Studio builds by default (the one you get by running `wingetdev`) has a different version number than normal winget, but it's just a placeholder value.

Long story short, the winget you have installed from the Microsoft Store shouldn't have additional features/functionality that the master branch doesn't have. It doesn't rely on any functionality (that I'm aware of) in App Installer, so missing those bits doesn't get you an inaccurate build.

**Comment by OfficialEsco** [comment]
https://github.com/microsoft/winget-pkgs/pull/35219#issuecomment-987116714

This should be the one they dropped the GitHub Release for:
Stable: 
```
Windows Package Manager v1.1.12986
Windows: Windows.Desktop v10.0.22000.348
Package: Microsoft.DesktopAppInstaller v1.16.12986.0
```
Preview:
```
Windows Package Manager (Preview) v1.2.3411-preview
Windows: Windows.Desktop v10.0.22000.348
Package: Microsoft.DesktopAppInstaller v1.17.3411.0
```
However, my win11 vm just got this stable release
```
Windows Package Manager v1.1.13405
Windows: Windows.Desktop v10.0.22000.348
Package: Microsoft.DesktopAppInstaller v1.16.13405.0
```
